app/views/spree/admin/variants/volume_prices.html.erb

```diff
@@ -1,24 +1,24 @@
-<%= render 'spree/admin/shared/product_tabs', current: Spree.t(:volume_pricing) %>
+<%= render 'spree/admin/shared/product_tabs', current: I18n.t('spree.volume_pricing') %>
 <%= render 'spree/shared/error_messages', target: @variant %>
 
 <% content_for :page_actions do %>
   <span id="new_add_volume_price" data-hook>
-    <%= button_link_to Spree.t(:add_volume_price), 'javascript:;', {
-      :'data-target' => 'tbody#volume_prices', class: 'btn-success spree_add_fields', id: 'add_volume_price'} %>
+    <%= link_to I18n.t('spree.add_volume_price'), 'javascript:;', {
+      :'data-target' => 'tbody#volume_prices', class: 'btn-success spree_add_fields', id: 'add_volume_price' } %>
```

Comment:
We need to add the `btn` class as well otherwise this will not be displayed as button anymore (Solidus >= 2.4 uses Bootstrap classes for buttons). Also we should use the primary button color instead of a green success button, as this is the default button color in Solidus 

```suggestion
      :'data-target' => 'tbody#volume_prices', class: 'btn btn-primary spree_add_fields', id: 'add_volume_price' } %>
```

Reply:
These changes can be tackled in a subsequent PR (that I already prepared locally). There are more admin UI changes necessary to update this extension to Solidus 2.4 and beyond.